packages/lib/src/vis/shared/LineSelectionMesh.tsx

```diff
@@ -1,9 +1,27 @@
+import type { Color } from '@react-three/fiber';
+
+import type { SelectionMeshProps } from '../models';
 import SelectionLine from './SelectionLine';
-import type { SelectionMeshProps } from './SelectionMesh';
 import SelectionMesh from './SelectionMesh';
 
-function LineSelectionMesh(props: SelectionMeshProps) {
-  return <SelectionMesh selectionComponent={SelectionLine} {...props} />;
+interface Props extends SelectionMeshProps {
```

Comment:
You could keep `SelectionMeshProps` in `SelectionMesh` with the `children` property (and `export type { Props as SelectionMeshProps };`, and have `Omit<SelectionMeshProps, 'children'>` here maybe. Same result, so up to you.